src/mixins/global.js

```diff
@@ -0,0 +1,45 @@
+import { mapState } from 'vuex';
+import Vue from "vue";
+
+export default {
+  methods: {
+    toUSD(avax) {
+      if (this.avaxPrice) {
+        return avax * this.avaxPrice;
+      }
+    },
+    toHex(dec) {
+      return '0x' + dec.toString(16);
+    },
+    toDec(hex) {
+      return parseInt(hex, 16);
+    },
+    async handleTransaction(fun, args, variable) {
```

Comment:
I think "variable" is not a good name for a variable ;)
I would rename it to something like "nameOfWaitingVariable"